docs/UserGuide.md

```diff
@@ -107,7 +107,7 @@ Bridal Boss is a **desktop app for managing contacts, optimized for use via a  L
 #### Email Addresses
 
 - **Format**:
-    - Must be in the form `local-part@domain`.
+    - Must be in the form `local-part@domain.com`.
```

Comment:
Email regex does not correspond to ending with .com, perhaps we can say end with xx, where x is an  alphanumeric character?

Reply:
latest commit should be more clear